source/drivers/driver-compatibility-reference.txt

```diff
@@ -718,6 +786,14 @@ Language Compatibility
      - PHP 7.0
      - HHVM 3.9
 
+   * - mongodb-1.2
+     -
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
```

Comment:
Yes, 3.15 is the latest LTS. HHVM 3.12 was also an LTS release, and is also supported. 3.13 and 3.14 don't have to be mentioned, although they do work. 

The HHVM driver works with HHVM 3.12 and 3.15, and I'm not sure where 3.9 came from, as I think we never supported that.